### ISSUE TITLE
Typography mixin causes issues with GOV.UK Panel

### ISSUE THREAD

**Issue by aliuk2012** [HMPO/hmpo-components#203]
The following code causes issues with GOV.UK Panel component https://github.com/HMPO/hmpo-components/blob/master/assets/stylesheets/_typeography.scss#L4-L30 as it set the font colour to black. This means the component is rendered with black text on a green background and causes accessibility issue with the contrast ratio.

### Incorrect
<img width="745" height="184" alt="Image" src="https://github.com/user-attachments/assets/47b2bd1b-7d63-4043-b558-c7638f306e87" />

### Correct

<img width="698" height="180" alt="Image" src="https://github.com/user-attachments/assets/38c3cd7e-1cd1-44a7-a9f5-b490a6dccbb0" />

This has caused an issue with at least two FCDO services that I'm aware of who have had to find work arounds.

In my service, I created a new class

```scss
// Title: HMPO Panel Component Title not the intended white text on green background
// Issue: HMPO Components has some mixins that force GOV.UK heading styling on default h1/h2/h3 etc. 
// As HMPO stylesheets are imported after GOV.UK stylesheets, the HMPO mixins override the GOV.UK heading styling
// and makes the Panel Components title black on green background instead of the intended white text on green background.
// This causes an accessibility issue with the contrast ratio being too low.
// https://github.com/HMPO/hmpo-components/blob/c8f6d336e27fb16b26b0c82ed28bcac4664b863e/assets/stylesheets/_typeography.scss#L4-L30
.app-panel__title h1{
  color: inherit;
}
```

In the view I have the following:

```
  {{ govukPanel({
      titleText: "Registration complete",
      classes: 'app-panel__title'
  }) }}
```
